PlusEnvironment.cs

```diff
@@ -127,6 +139,10 @@ public async Task<bool> Start()
             foreach (var task in _startableTasks)
                 await task.Start();
 
+            await _achievementManager.Init();

```

Comment:
Implement the `IStartable` interface in the `AchievementManager` so manually calling init here is not needed.

Reply:
Done